packages/core/lib/flutter_generator.dart

```diff
@@ -56,8 +56,11 @@ class FlutterGenerator {
     }
 
     if (flutterGen.colors.enabled && flutterGen.colors.inputs.isNotEmpty) {
-      final generated =
-          generateColors(pubspecFile, formatter, flutterGen.colors);
+      final generated = generateColors(
+        pubspecFile,
+        formatter,
+        flutterGen.colors,
+      );
```

Comment:
I dont think you need to update this call formatter?

Reply:
I think so, I have updated this code.